src/components/HeaderSection/AccountMenu/AccountMenu.tsx

```diff
@@ -0,0 +1,102 @@
+import Logout from '@mui/icons-material/Logout';
+import Box from '@mui/material/Box';
+import Divider from '@mui/material/Divider';
+import IconButton from '@mui/material/IconButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Tooltip from '@mui/material/Tooltip';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import WelcomeHeader from 'src/components/ProfilePage&HeaderSection/WelcomeHeader';
+
+import logout from '../../../utils/auth/logout';
+import UserAvatar, { AvatarSize } from '../../ProfilePage&HeaderSection/UserAvatar';
+
+export default function AccountMenu() {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  return (
+    <>
+      <Box sx={{ display: 'flex', alignItems: 'center', textAlign: 'center', marginRight: '3rem' }}>
+        <Tooltip title="Account settings">
+          <IconButton
+            onClick={handleClick}
+            size="small"
+            sx={{ ml: 2 }}
+            aria-controls={open ? 'account-menu' : undefined}
+            aria-haspopup="true"
+            aria-expanded={open ? 'true' : undefined}
+          >
+            <UserAvatar name="M" size={AvatarSize.Small} />
+          </IconButton>
+        </Tooltip>
+      </Box>
+      <Menu
```

Comment:
I think it could be nice if the circle with an avatar had `cursor: pointer` when the menu is open.

Reply:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/80357638/163173940-fcc0ad08-373a-43a2-9a75-dde80554d629.png">
It's a default MUI behavior :)